src/Integration/Flysystem/S3/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## NOT RELEASED
 
+## 1.0.1
+
+### Fixed
+
+- Fixed bucket used in `copy` calls
```

Comment:
Removed `urlencode()`  and added bucket to `CopySource` in the `copy()` method. This comply with the Flysystem interface and the AWS adapter.